src/commands/project/create.js

```diff
@@ -0,0 +1,79 @@
+const { flags } = require('@oclif/command')
+const { postProject } = require('../../requests/project')
+const {
+    getProjectIdentifierArg,
+    splitPathParams,
+    splitFlagParams
+} = require('../../support/command/parse-input')
+const BaseCommand = require('../../support/command/base-command')
+
+class CreateProjectCommand extends BaseCommand {
+
+    async run() {
+        const { args, flags } = this.parse(CreateProjectCommand)
+        const projectPath = getProjectIdentifierArg(args)
+        const [owner, projectName] = splitPathParams(projectPath)
+        const body = {
+            'owner': owner,
+            'name': projectName
+        }
+        // Append optional flags to body if present
+        if (flags.description !== undefined) body.description = flags.description
+        if (flags.apis !== undefined) body.apis = splitFlagParams(flags.apis)
+        if (flags.domains !== undefined) body.domains = splitFlagParams(flags.domains)
+
+        await this.createProject(owner, projectName, JSON.stringify(body))
+    }
+
+    async createProject(owner, projectName, body) {
+        const createRequest = {
+            pathParams: [owner],
+            body: body
```

Comment:
same as above, `body` is enough due to es6 object naming conversions

Reply:
Changed here too